WooCommerce/Classes/ViewRelated/Dashboard/DashboardCustomization/DashboardCustomizationView.swift

```diff
@@ -15,7 +15,7 @@ struct DashboardCustomizationView: View {
             MultiSelectionReorderableList(contents: $viewModel.allCards,
                                           contentKeyPath: \.name,
                                           selectedItems: $viewModel.selectedCards,
-                                          inactiveItems: viewModel.inactiveCards,
+                                          inactiveItems: [], // Intentionally empty to not show inactive cards.
```

Comment:
Technically do not need to be done since `[]` is a default value, but explicitly saying this to make the intention clear.